ui.apps/src/main/content/jcr_root/apps/venia/components/structure/header/v1/header/clientlibs/css/header-component.css

```diff
@@ -44,6 +44,13 @@
     color: rgb(var(--venia-teal));
 }
 
+.header {
+    position: -webkit-sticky;
+    position: sticky;
+    top: 0;
+    z-index: 10;
+}
+
 .header__root {
```

Comment:
I'm not a css expert, but adding replacing `position: fixed` and adding `width: 100%` to `header__root` also fixes that problem

Reply:
It seems "sticky" is useful when you want something to scroll for a while an then at some point stick ... but hey I'm really not a css guru!

Reply:
A "sticky" element takes space on the page and the rest of the elements are positioned accordingly. A "fixed" element doesn't take space on the page and it remains in that position. 
However, I see Venia uses "sticky" as well so we might as well go with that.